include/megaapi_impl.h

```diff
@@ -2983,6 +2983,10 @@ class MegaApiImpl : public MegaApp
         std::mutex mSyncable_fa_mutex;
 #endif
 
+        void syncput_result(error, handle) override;
+        void syncputheartbeat_result(error) override;
+        void syncremove_result(error, handle) override;
```

Comment:
We could use some comments. here, or probably in megapp.h: one line indicating what does this do.
Plus, having parameters with names (surronded by /* */) would also help: 
e.g:
```
        // sync registered / updated 
        void syncput_result(error, handle /*syncId*/) override;
```

Reply:
Yes, it's ok for me. I'll modify it